.sphinx/_templates/header.html

```diff
@@ -28,9 +27,20 @@
             <a href="{{ github_url }}" class="p-navigation__sub-link p-dropdown__link">GitHub</a>
           </li>
 
+        </ul> 
+
+      </li>
+      <li>
+        {% set contributors = get_contribs(github_url, github_folder, pagename, page_source_suffix) %}
+        <a href="#" class="p-navigation__link nav-more-links_cont">{{ contributors |length}} - Contributors</a>
```

Comment:
Before using the `contributors` variable, check if it actually has a value. If something went wrong with retrieving the information and `contributors` is `None`, we should leave out the whole section (instead of displaying "0 contributors").